setup.cfg

```diff
@@ -130,6 +130,7 @@ testing =
     IPython>=7.25.0
     qtconsole>=4.5.1
     rich>=12.0.0
+    napari-plugin-manager @ git+https://github.com/napari/napari-plugin-manager.git
```

Comment:
With this change nightly builds for conda forge will not be working as the package is not available from conda-forge. 

What is preventing us from releasing it?

Reply:
Nothing so far. I am just bootstrapping it like this to make sure everything works before we release something that doesn't work. Not that it will break anything, but just keeping the release history clean if we can. If you are happy with the results here, I can cut a beta.

Reply:
I prefer to go with beta. Also similar to `app-model` I like to set an upper bound in requirements to have more freedom when updating the plugin dialog. If there will be a need to add breaking change, it will be just possible to bump the version above the upper bound to not break the installation of older napari releases 
(of course it such a condition will be true after release 0.5.0).

Reply:
What kind of version upper bound were you thinking of, @Czaki? I published 0.1.0a0 so I guess `<0.2.0`? Feel free to add your suggestions here.

Reply:
Yes. I think that 0.2.0 may be good.

Reply:
```suggestion
    napari-plugin-manager <0.2.0
```

Reply:
```suggestion
    napari-plugin-manager <0.2.0
```